anti.md

```diff
@@ -11,34 +11,65 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The main kind of person using this language is someone who wants to check up on
+many different websites all in one place. Right now they have to individually
+load up each different site and see if there's anything new, but with a
+language to help them they could see everything in one place. With this
+language they could just input what they care about and it would show them what
+they want to see.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL could be appropriate because it would specify what the user cares about.
+Unfortunately, I don't think it would work because websites have so many
+different formats that trying to incorporate all of them would be a nightmare.
+A standard user would probably want text posts, images, and videos all in the
+same place, and to do this a large part of the Internet would need to be
+scraped and parsed to get them what they want.
```

Comment:
I don't think this is the reason that a DSL isn't a way to solve this problem, but rather an argument that the problem itself isn't solvable.